backend/fms_core/migrations/0019_v3_1_2.py

```diff
@@ -0,0 +1,65 @@
+from django.conf import settings
+from django.db import migrations
+from django.contrib.auth.models import User
+import reversion
+import datetime
+
+ADMIN_USERNAME='biobankadmin'
+
+
+def fix_process_data(apps, schema_editor):
+    Version = apps.get_model("reversion", "Version")
+    Revision = apps.get_model("reversion", "Revision")
+    ProcessSample = apps.get_model("fms_core", "ProcessSample")
+    Process = apps.get_model("fms_core", "Process")
+
+    affected_revisions_ids = Revision.objects.filter(date_created__range=["2021-04-29", "2021-05-01"]).values_list('id', flat=True)
+
+    with reversion.create_revision(manage_manually=True):
+        reversion.set_comment("Curation of multiple processes into one process per template")
+        reversion.set_user(User.objects.get(username=ADMIN_USERNAME))
+
+        for revision_id in affected_revisions_ids:
+            # Getting all ProcessSample created in this revision
+            affected_process_samples_ids = Version.objects.filter(serialized_data__contains='"model": "fms_core.processsample"',
```

Comment:
Likely much faster to use the id of the content_type_id/model instead of matching the serialized data to a string.

Reply:
Even though it might be slightly faster (btw the request also filters the revision_id), I think it's safer to filter by serialized data content, than by giving it a fixed content_id (i.e. '6'). Also, we dont have performance issues here at all, so I dont think it's necessary.

Reply:
you can also use the name of the model to filter using the same id (you can see references in past migrations) ... I agree that performance is not a problem here, but you should still use the most efficient way whenever you can. We tend to get used to using a certain way of doing things ... at some point it might matter.

Reply:
Referring the model name seems good to me! Just didnt want to use the plain id number hardcoded :P

Reply:
> content_type__model='processsample'